setup.py

```diff
@@ -76,6 +76,7 @@ def get_requirements():
 setup(
     name=COMMAND_NAME,
     version=VERSION,
+    requires_python='>= 3.9',
```

Comment:
I suggest 3.7 as min.

Reply:
Are there any systems we run on that use this? Iirc the consensus of the meeting yesterday was to use whatever Rocky Linux 9 ships, which is 3.9.

Reply:
https://github.com/simplyblock-io/sbcli/blob/e73abc3ade9b7c519c866fe5edbb329f72e6dcab/docker/Dockerfile_base#L1-L2

We've have been using python3.9 for our internal testing so far. So I'd agree with having 3.9 as the minimum python version.